Gemfile

```diff
@@ -46,8 +46,10 @@ end
 
 # Everything except AIX and Windows
 group(:ruby_shadow) do
-  # if ruby-shadow does a release that supports ruby-3.0 this can be removed
-  gem "ruby-shadow", git: "https://github.com/chef/ruby-shadow", branch: "lcg/ruby-3.0", platforms: :ruby unless RUBY_PLATFORM == "x64-mingw-ucrt"
+  install_if -> { RUBY_PLATFORM != "x64-mingw-ucrt" } do
+    # if ruby-shadow does a release that supports ruby-3.0 this can be removed
+    gem "ruby-shadow", git: "https://github.com/chef/ruby-shadow", branch: "lcg/ruby-3.0", platforms: :ruby unless RUBY_PLATFORM == "x64-mingw-ucrt"
```

Comment:
Looks like there's `install_if` and `unless` both conditions doing same thing twice. Get rid of one of those?

Reply:
Argh